docs/developer/guidelines/dashboards.md

```diff
@@ -54,128 +49,115 @@ Ensure logos render correctly on the Dashboard List page and within the preset d
 
-There are different types of metric widgets. The most commonly used are [timeseries](https://docs.datadoghq.com/dashboards/widgets/timeseries/),
- [query values](https://docs.datadoghq.com/dashboards/widgets/query_value/), and [tables](https://docs.datadoghq.com/dashboards/widgets/table/).
+5. If you’re showing a legend, make sure the aliases are easy to understand
 
-For more information on the available widget types, see the [list of supported dashboard widgets](https://docs.datadoghq.com/dashboards/widgets/).
+6. Graph titles should summarize the queried metric. Do not indicate the unit in the graph title because unit types are displayed automatically from metadata. An exception to this is if the calculation of the query represents a different type of unit.
 
-!!! note
-    Metric widgets usually have a timeframe of 4 hours. Query value widgets have the timeframe of 1 hour.
+### QA
 
-##### Timeseries
+1. Always check a dashboard at 1280px wide and 2560px wide to see how it looks on a smaller laptop and a larger monitor. The most common screen widths for dashboards are 1920, 1680, 1440, 2560, and 1280px respectively. They make up more than half of all dashboard page views combined.
 
-Timeseries widgets allow you to visualize the evolution of one or more metrics over time.
+![QA at different widths](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/qa_widths.png)
 
-###### Color Palette
+!!! tip
+    Use [`TV Mode`](https://docs.datadoghq.com/dashboards/screenboards/#tv-mode) to ensure that the whole dashboard fits your screen.
 
-| Types of metrics | Palette |
-| ---------------- | ------------- |
-| Errors/negative (e.g. Queued requests) | Warm or orange color |
-| Memory | Cool or blue color |
-| Read | Green color |
-| Write | Purple color |
+### Visual Style
 
-###### Display type
+1. Format notes to make them fit their use case. Try the presets “caption”, “annotation”, or “header”, or pick your own combination of styles. Avoid using the smallest font size for notes that are long or include complex formatting, like bulleted lists or code blocks.
 
-| Types of metrics | Display Type |
-| ---------------- | ------------- |
-| Volume (e.g. Number of connections) | `area` |
-| Counts (e.g. Number of errors) | `bars` |
-| Multiple groups or default | `lines` |
+![Note styles](https://raw.githubusercontent.com/DataDog/integrations-core/master/docs/developer/assets/images/note_styles.png)
 
+2. Use colors to highlight important relationships and to improve readability, not for style. If several groups are related, apply the same group header color to all of them. If you’ve applied a green header color to a group, try making its notes green as well. If two groups are related, but one is more important, try using the “vivid” color on the important group and the “light” color on the less important group. Be careful not to overuse color e.g. don’t make every group on a dashboard vivid blue. 
```

Comment:
suggestion: also include staying away from greys/muted yellow-tones in notes, as they come off as less bold/eye-catching and users may be less inclined to read what they say

Reply:
Added